datastructures/dynamic-array/dynamicarray.go

```diff
@@ -1,4 +1,4 @@
-package dynamic-array
+package dynamic_array
```

Comment:
This has been changed to the `package dynamicarray` for better consistency.

Maybe pull this file from the master branch of this repository to your local and commit the file without any changes